tests/functions.sh

```diff
@@ -727,12 +727,16 @@ function set_nic_sg {
         id: \"nic-$nic_id\"
         ip: \"42.0.0.$nic_id\"
         ip_anti_spoof: true
-        security_group: \"$sg\"
       }
     }
   }
 }
 " > $f
+
+    for i in $sg_list; do
+	sed -i "s/ip_anti_spoof: true/& \n \tsecurity_group: \"$i\"/g"  $f
+    done
+
     request $but_id $f
 }
```

Comment:
Can't work with more that 2 sg. That's to complicated !

You can simply do something like: 
```
echo "...
[...]
ip_anti_spoof: true
" > $f

for sg in $sg_list; do
    echo $sg >> $f
done

echo "
       }
     }
   }
 }
" >> $f
```